packaging/flatpak/com.freerdp.FreeRDP.json

```diff
@@ -37,6 +37,7 @@
 		"--socket=cups",
 		"--socket=wayland",
 		"--filesystem=xdg-download",
+		"--talk-name=com.microsoft.identity.Broker1",
```

Comment:
Hi, sorry for the late response. There's a little typo which currently breaks it: This must be: `com.microsoft.identity.broker1`. I just changed that locally and now it works!

Anyways, many thanks for making this possible!

Reply:
ok, copied from here: https://github.com/siemens/sso-mib/blob/main/dbus/spec/com.microsoft.identity.broker1.xml 
is this case difference expected?

Reply:
DBus and their names... What is in the spec is the interface name. What we need to allow is the name on the bus, which is different in this case.
![image](https://github.com/user-attachments/assets/2fc984a5-41c4-4c50-9448-5f7f58bdc9db)